### DISCUSSION TITLE
how can i shift add button in list type below the elment instead of top and it should dynamically shift  below the last element

### DISCUSSION THREAD

**Discussion by vyombriq** [clientIO/joint#2806] [Q&A]
### Introduction

![image](https://github.com/user-attachments/assets/fd392954-303a-4477-823f-e031828541f9)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
That is not currently supported.

You might try to re-append the buttons one the inspector is rendered:
```ts
inspector.on('render', => {
  Array.from(inspector.el.querySelectorAll('.btn-list-add')).forEach(buttonEl => buttonEl.parentNode.appendChild(buttonEl))
});
```
